data/src/main/java/me/androidbox/data/local/dao/EventDao.kt

```diff
@@ -0,0 +1,25 @@
+package me.androidbox.data.local.dao
+
+import androidx.room.*
+import kotlinx.coroutines.flow.Flow
+import me.androidbox.data.local.DatabaseConstant
+import me.androidbox.data.local.entity.EventEntity
+
+@Dao
+interface EventDao {
+    @Query("SELECT * FROM ${DatabaseConstant.EVENT_TABLE} where dateCreated = :selectedDate ORDER BY dateCreated ASC")
+    fun getEvent(selectedDate: Long): Flow<List<EventEntity>>
```

Comment:
Would this be correct if we are going to query the event on the date it was created?

Reply:
Explained that in WhatsApp